### ISSUE TITLE
Add way to refer to specific task results on others

### ISSUE THREAD

**Issue by joaomdmoura** [joaomdmoura/crewAI#106]
There would be value on being able to refer to a specific task result on other tasks down the road instead of just relying on the outcome from the previous

**Comment by greysonlalonde** [comment]
@joaomdmoura  leveraging the default [Task IDs](https://github.com/joaomdmoura/crewAI/blob/4f22bbf4d4b886e95bd3bbd4b35b8e49171bad72/src/crewai/task.py#L26) could be a good base for this 🤗

**Comment by joaomdmoura** [comment]
Indeed! Now as tasks get completed their outcome is added to task.output so that is also in there
